### PR TITLE
{cae}[foss/2020b] OpenFOAM vv2206

### PR DESCRIPTION
**Pull request by DirectorSloan**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
Test report by @sassy-crick
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
hp - Linux Debian GNU/Linux 11, x86_64, 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz, Python 3.9.2
See https://gist.github.com/ef1b8adf9feaa06ed08199ac322b281f for a full test report.

**Comment by sassy-crick** [comment]
I would like to install that on our cluster. Is there a way that one of the maintainers could have a look at it please? Thanks!

**Comment by DirectorSloan** [comment]
Dear easybuilders contribution team. What can I do to support the merge of this easyconfig? best regards

**Comment by SebastianAchilles** [comment]
Apologies for this PR going stale. Let's resolve the merge conflict and then generate a few more test reports.

The `OpenFOAM-v2206-cleanup.patch` that was added in https://github.com/easybuilders/easybuild-easyconfigs/pull/16947 is almost identical to the patch from this PR, but also includes support for KaHIP. That patch seems to also work for this build. Therefor I would suggested to reuse the patch that was already merged.

File: easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v2206-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+source_urls = ['https://sourceforge.net/projects/openfoam/files/%(version)s/']
+sources = [SOURCE_TGZ]
+patches = [
+    ('OpenFOAM-v2206-cleanup.patch', 1),
+    'OpenFOAM-v2206-wmake-ompi.patch',
+]
+checksums = [
+    'db95eda4afb97ca870733b2d4201ef539099d0778e3f3eca9a075d4f1a0eea46',  # OpenFOAM-v2206.tgz
+    '8a24b579d8e9d645cad8a64bdd2841bd3c5cb3ff982ce829af2017a297c4b5ae',  # OpenFOAM-v2206-cleanup.patch
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v2206-foss-2020b.eb:26]:
```suggestion
    '25333124581acae57c173587de4ebd6e143b894b1a26e4f0326db8b7e0cb1972',  # OpenFOAM-v2206-cleanup.patch
```

**Comment by SebastianAchilles** [comment]
@DirectorSloan I am not able to update your PR. Can you please merge the latest change from develop that include the patch that was already merged? E.g. by running
```
eb --sync-pr-with-develop=16125
```

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**